tools/src/publish-packages/helpers.ts

```diff
@@ -230,13 +230,16 @@ export function resolveReleaseTypeAndVersion(parcel: Parcel, options: CommandOpt
     ReleaseType.PATCH
   );
   const allVersions = pkgView?.versions ?? [];
+  const isOnSDKBranch = (await Git.getSDKVersionFromBranchNameAsync()) != null;
 
   if (prerelease) {
     // Make it a prerelease version if `--prerelease` was passed and assign to the state.
     state.releaseType = ('pre' + highestReleaseType) as ReleaseType;
   } else if (getPrereleaseIdentifier(pkg.packageVersion)) {
     // If the current version is a prerelease, just increment its number.
     state.releaseType = ReleaseType.PRERELEASE;
+  } else if (isOnSDKBranch) {
+    state.releaseType = ReleaseType.PATCH;
```

Comment:
Can we warn users that this would normally be a `highestReleaseType` but because it's inside an SDK branch then it's a patch?